docs/user_guide/02_hybrid_queries.ipynb

```diff
@@ -142,9 +139,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 26,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "@credit_score:{high}=>[KNN 10 @user_embedding $vector AS vector_distance] RETURN 6 user credit_score age job office_location vector_distance SORTBY vector_distance ASC DIALECT 2 LIMIT 0 10\n"
```

Comment:
Looks like something is logging out the query string by default? I do like showing this sometimes in our notebooks. But if this happens on every `.query()` might be too noisy.